src/figma/tokens.json

```
@@ -1427,6 +1487,16 @@
           "value": "#FFD33D80",
           "description": "(yellow500: #FFD33D 50% opacity) [DEPRECATED] warning.disabled should be used for all disabled component colors such as buttons",
           "type": "color"
+        },
+        "default-hover": {
+          "value": "#FFC60A",
+          "type": "color",
+          "description": "(#FFC60A): For warning with stronger contrast."
+        },
+        "default-pressed": {
+          "value": "#FFEAA3D1",
```

Remove the opacity from the hex
```suggestion
          "value": "#FFEAA3",
```